Sources/SwiftRedis/RedisResp.swift

```diff
@@ -22,10 +22,10 @@ enum RedisRespStatus {
     case connected
 }
 
-class RedisResp {
+public class RedisResp {
     ///
     /// Socket used to talk with the server
-    private var socket: Socket?
+    public var socket: Socket?
```

Comment:
Why do you need to change the visibility?

Reply:
See above.

Reply:
Nothing is public inside, so the user can only access the "raw" socket, not calling, eg. `disconnect`.